packages/protocol/contracts/ZkAsset/ZkAssetMintableTest.sol

```diff
@@ -0,0 +1,27 @@
+pragma solidity >=0.5.0 <0.6.0;
```

Comment:
I'm not a big fan of this pattern, especially not if it means having test contracts in a non-test directory. I understand that you're making sure that a contract can use confidentialTransferFrom, but I don't see what that teaches us compared to say having address[2] call these methods after being approved?

Reply:
Yeah good point, that's a nicer way to do it - will update the tests.

We clean away this pattern of having test contracts in non-test directories on `develop`